src/features/posts/components/add-post-form.tsx

```diff
@@ -1,38 +1,62 @@
 import { useState } from 'react';
+import { useAuth } from '../../../shared/hooks/useAuth';
 import { PostContent } from '../types/post-content';
 import { postContentSchema } from '../utils/validate';
 import { fromZodError } from 'zod-validation-error';
-
+import { Editor } from '@tinymce/tinymce-react';
 import Box from '@mui/material/Box';
 import TextField from '@mui/material/TextField';
 import { Button } from '@mui/material';
+import { ApiKeyTinyMMC } from '../../../shared/config/confidential-data';
+import { useCreatePost } from '../api/use-create-post';
+import { generateID } from '../utils/generate-id';
 
-import ReactQuill from 'react-quill';
-import 'react-quill/dist/quill.snow.css';
-
+import { ref, uploadBytes } from "firebase/storage";
+import { imageDatabase } from '../../../shared/config/firebase-image';
 
-let DEFAULT_POST: PostContent = {
+const DEFAULT_POST: PostContent = {
+    postId: 0,
+    userId: 0,
     title: '',
     content: ''
 }
 
 export const AddPostForm = () => {
     const [content, setContent] = useState<string>('');
     const [error, setError] = useState<PostContent>(DEFAULT_POST);
-    const handleContentChange = (content: string) => setContent(content);
+    const [image, setImage] = useState<any>(null);
+    const { userData } = useAuth();
+    const { mutate } = useCreatePost()
+
+    const handleContentChange = (e: React.FormEvent<HTMLFormElement> | any) => setContent(e.target.getContent());
+    const randomID = generateID(1000000000);
```

Comment:
Troche dziwne, że po stronie frontendu tworzysz ID dla postu, tym powinien się zajmować backend

Reply:
Tak było na początku, ale zrobiłem to ze względu na to żeby takie samo id miało zdjęcie w firebase. Możliwe jak bym wypchnął tego firebasa do backendu to pozbył bym się tego problemu.